src/scss/03_organisms-blocks/form/_inner.scss

```diff
@@ -1,9 +1,15 @@
+// stylelint-disable media-feature-name-whitelist
+
 .form {
   &__inner {
     background: $color-alt;
     height: 100%;
     padding: 15px;
 
+    @media (max-width: 767px) {
+      padding: 25px;
+    }
+
```

Comment:
Not sure if this would work but couldn't the values be switched and the `max-width` turned into a `min-width`?